### PR TITLE
heading format for sections, concept topics, procedure topics

### PR DESCRIPTION
**Pull request by lamagnifica**
@pdesjardins

### CODE REVIEW

**Comment by edx-webhook** [comment]
## [DOC-XXXX](https://openedx.atlassian.net/browse/DOC-XXXX)

Add a description of your changes.


### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give
reviewers guidance on turnaround time.


### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer
finishes and gives :+1:. 

- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check/copy edit/dev edit): 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.


### Testing

- [ ] Ran ./run_tests.sh without warnings or errors


### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

 
### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here


### Post-review

- [ ] Add description to release notes task as a comment
- [ ] Squash commits

**Comment by lamagnifica** [comment]
@pdesjardins is this description clearer?

File: en_us/edx_style_guide/source/preferred.rst [hunk 1]
```diff
@@ -82,10 +82,10 @@ Preferred Usage
       Dashboard." If necessary, explain the file type. For example: "A Word
       (.doc) file."
   * - Headings
-    - For sections that contain concepts and procedures, use a gerund to start
-      the title. For concepts, use a gerund to start the title. For
-      procedures, use an imperative verb to start the title. Do not use nouns
-      or noun phrases.
+    - Use a gerund to start the title of top level section headings. For topics
+      that introduce a concept, use a gerund to start the title. For topics
```
Comment (pdesjardins) [inline en_us/edx_style_guide/source/preferred.rst:86]: This makes sense. I'm understanding that procedures should not be at the top level of a document (a top level section). Is that what you meant? Also, "topics that introduce a concept" means sections, at any level, that cover conceptual information, right? If so, maybe replace "topics" with "sections at any level" in this and in the following sentence about procedures.

File: en_us/edx_style_guide/source/preferred.rst [hunk 2]
```diff
@@ -82,10 +82,10 @@ Preferred Usage
...
+      that describe a procedure, use an imperative verb to start the title. Do
+      not use nouns or noun phrases.
```
Comment (pdesjardins) [inline en_us/edx_style_guide/source/preferred.rst:88]: I think we should describe how to title sections that contain reference information. That seems like a natural fit for nouns and noun phrases. For example, "Alphabetical Event List" and "Course Team Events."

File: en_us/edx_style_guide/source/preferred.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/edx_style_guide/source/preferred.rst:86]: this makes me realize that our arbitrary decision to replace the notion of "chapter" with "section", and to call the most granular heading-level info a "topic", might not be perfectly obvious...
...so all of our guides are made up of sections, which should be introduced with a heading that begins with a gerund.
within a section, we include both procedural and conceptual topics. Procedures get the imperative headings, concepts get the gerunds.

Comment (lamagnifica) [inline en_us/edx_style_guide/source/preferred.rst:86]: Agreed in principle. The team decided that we aren't going to specifically call out "reference topics" as any different from conceptual topics, however. I would like to revisit that decision.

**Comment by pdesjardins** [comment]
This definitely clarifies. :+1: I think we should discuss reference headings as a group. I wouldn't want to try gerundizing "course team events."

**Pull request merged**